### PR TITLE
use generic Output type annotations in hacker news

### PR DESCRIPTION
**Pull request by sryza**
## Checklist

- [ ] My change requires a change to the documentation and I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by OwenKephart — APPROVED** [review]
oh cool I didn't know this was a thing

**Comment by OwenKephart** [comment]
> 

oh it is not

**Comment by dpeng817** [comment]
This should be landable now!

**Review by dpeng817 — CHANGES_REQUESTED** [review]
To your queue to verify everything works

File: examples/hacker_news/hacker_news/ops/id_range_for_time.py [hunk 1]
```diff
@@ -87,20 +87,16 @@ def _get_item_timestamp(item_id):
...
-@op(
-    required_resource_keys={"hn_client", "partition_bounds"},
-    out=Out(
-        Tuple[int, int],
-        description="The lower (inclusive) and upper (exclusive) ids that bound the range for the partition",
-    ),
-)
-def id_range_for_time(context):
+@op(required_resource_keys={"hn_client", "partition_bounds"})
+def id_range_for_time(context) -> Output[Tuple[int, int]]:
```
Comment (dpeng817) [inline examples/hacker_news/hacker_news/ops/id_range_for_time.py:91]: you're using the wrong tuple here

**Review by dpeng817 — CHANGES_REQUESTED** [review]

**Pull request merged**